sys/cheri/cheri.h

```diff
@@ -114,6 +114,11 @@ extern void * __capability swap_restore_cap;
 /* Root of all sealed kernel capabilities. */
 extern void * __capability kernel_root_sealcap;
 
+#ifdef __CHERI_PURE_CAPABILITY__
+/* Root kernel capability */
+extern void * __capability kernel_root_cap;
+#endif
```

Comment:
```suggestion
#ifdef __CHERI_PURE_CAPABILITY__
/* Root kernel capability */
extern void * __capability kernel_root_cap;
#endif

/* Root of all sealed kernel capabilities. */
extern void * __capability kernel_root_sealcap;
```
Maybe there's a specific use for it in a hybrid kernel too, I can vaguely imagine there being one if you want to make DDC more restrictive (especially with a non-offsetting interpretation for DDC, which you can have on Morello if you set CCTLR_ELx.DDCBO to 0).

Reply:
For now it's only used in purecap, so I think leaving it under the #ifdef is sufficient.

Reply:
Yeah, deleting a few ifdefs is easy enough if we ever want it, I was just musing out loud whilst I was already commenting.